generator/internal/genclient/translator/protobuf/protobuf.go

```diff
@@ -100,11 +102,12 @@ func makeAPI(req *pluginpb.CodeGeneratorRequest) *genclient.API {
 			}
 			state.ServiceByID[service.ID] = service
 			for _, m := range s.Method {
-				method := &genclient.Method{}
-				method.HTTPInfo = parseHTTPInfo(m.GetOptions())
-				method.Name = m.GetName()
-				method.InputTypeID = m.GetInputType()
-				method.OutputTypeID = m.GetOutputType()
+				method := &genclient.Method{
```

Comment:
This change seem unrelated, but meh.

Reply:
This was more for readability, but happy to put in a separate PR in the future